package-lock.json

```diff
@@ -3530,6 +3530,15 @@
             "integrity": "sha512-gac8OEcQ2Li1dxIEWGZzsp2BitJxwkwcOm0zHAJLcPJaVvm58FRnk6RkuLRpU1EujipU2ZFODv2P9DLMfnV8mw==",
             "dev": true
         },
+        "adler-32": {
+            "version": "1.2.0",
+            "resolved": "https://registry.npmjs.org/adler-32/-/adler-32-1.2.0.tgz",
+            "integrity": "sha1-aj5r8KY5ALoVZSgIyxXGgT0aXyU=",
+            "requires": {
```

Comment:
@aparnaacco879 Remove package-lock.json

Reply:
Remove the package-lock file itself @aparnaacco879

Reply:
Done with the changes @naveenkumarkandasamy